packages/api/ecs/async-operation/package.json

```diff
@@ -20,9 +20,10 @@
     "test": "../../../../node_modules/.bin/ava"
   },
   "dependencies": {
-    "@cumulus/aws-client": "10.1.1",
-    "@cumulus/db": "10.1.1",
-    "@cumulus/logger": "10.1.1",
+    "@cumulus/aws-client": "10.1.0",
+    "@cumulus/db": "10.1.0",
+    "@cumulus/es-client": "10.1.2-alpha.0",
+    "@cumulus/logger": "10.1.0",
```

Comment:
```suggestion
    "@cumulus/aws-client": "10.1.1",
    "@cumulus/db": "10.1.1",
    "@cumulus/es-client": "10.1.2-alpha.0",
    "@cumulus/logger": "10.1.1",
```

Reply:
Hmm, yes those got overlooked in the 10.1.1 release.     Updating.

Reply:
The branch is protected, so updates here: https://github.com/nasa/cumulus/pull/2896

Reply:
More specifically a4429e206a5d04bfdc45e1e4520cf1a7ca9a184f

Reply:
Hmm I don't see the updates in that PR